### PR TITLE
Add faq on creating workflow test

### PR DESCRIPTION
**Pull request by hexylena**

### CODE REVIEW

File: faqs/gtn/gtn_workflow_testing.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   This tutorial has a workflow (`.ga`) and a test, notice the `-test.yml` that has the same name as the workflow `.ga` file.
+
+   ```
+   machinelearning/workflows/machine_learning.ga
+   machinelearning/workflows/machine_learning-test.yml
+   ```
+
+   You want to find tutorials without either of these.
+
+2. Check if it has a workflow (if it does, skip to step 5.)
```
Comment (simonbray) [inline faqs/gtn/gtn_workflow_testing.md:20]:
```suggestion
   You want to find tutorials without the `-test.yml` file. The workflow file might also be missing.

2. Check if it has a workflow (if it does, skip to step 5.)
```

File: faqs/gtn/gtn_workflow_testing.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+3. Follow the tutorial
+4. Build a workflow
```
Comment (simonbray) [inline faqs/gtn/gtn_workflow_testing.md:22]:
```suggestion
4. Extract a workflow from the history
```
Probably this is easier?

File: faqs/gtn/gtn_workflow_testing.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+   ![screenshot of the workflow invocation page. The user drop down shows where to find this page, and a red box circles a field named "Invocation ID"]({% link faqs/gtn/images/invocation.png %})
+
+7. Install the latest version of `planemo`
+
+   ```
+   # In a virtualenv
+   pip install git+https://github.com/galaxyproject/planemo
+   ```
+
+8. Run the command to initialise a workflow test
```
Comment (simonbray) [inline faqs/gtn/gtn_workflow_testing.md:35]:
```suggestion
8. Run the command to initialise a workflow test from the `workflows/` subdirectory - if it doesn't exist, you might need to create it first.
```

File: faqs/gtn/gtn_workflow_testing.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+   ```
+   $ tree
+   .
+   ├── test-data
+   │   ├── input dataset(s).shapefile.shp
+   │   └── shapefile.shp
+   ├── testing-openlayer.ga
+   └── testing-openlayer-tests.yml
+   ```
```
Comment (simonbray) [inline faqs/gtn/gtn_workflow_testing.md:51]: Quite likely this is too advanced for a quick FAQ, but you could mention replacing output comparisons with assertions?

Comment (hexylena) [reply faqs/gtn/gtn_workflow_testing.md:51]: No that's a useful thing to mention, let's add it.

Comment (hexylena) [reply faqs/gtn/gtn_workflow_testing.md:51]: Done! if you don't mind checking / approving if it's ok

**Review by shiltemann — APPROVED** [review]

**Pull request merged**